packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ISummaryConfiguration } from "@fluidframework/protocol-definitions";
+import { ILoaderOptions  } from "@fluidframework/container-definitions";
+import { IContainerRuntimeOptions, IGCRuntimeOptions, ISummaryRuntimeOptions } from "@fluidframework/container-runtime";
+import {
+    booleanCases,
+    generatePairwiseOptions,
+    OptionsMatrix,
+    numberCases,
+} from "@fluid-internal/test-pairwise-generator";
+import { Lazy } from "@fluidframework/common-utils";
+
+const loaderOptionsMatrix: OptionsMatrix<ILoaderOptions> = {
+    cache: booleanCases,
+    hotSwapContext: booleanCases,
+    provideScopeLoader: booleanCases,
+    maxClientLeaveWaitTime: numberCases,
```

Comment:
What are the values that will be used for these numberCases? Should a list of values be provided by the tests? Or will it be auto computed?
I am trying to understand what will the flow look like when using these to write tests. Maybe a simple example of this in the description? Or an example test (I am guessing this will be coming in a follow up PR anyway).

Reply:
this already integrates with the stress test. right now, number cases is just [undefined], as numbers aren't interesting dimensions to explore, as least not our current numbers. In general all these values are the lists of values the test want to explore and that will be expanded when generatePairwiseOptions is called. i plan to add some unit test that might make it a bit more clear

Reply:
sounds good.